plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/revisited/ModulesManagerTest.java

```diff
@@ -300,6 +298,25 @@ public void testGetAllModulesStartingWithRemoval() {
         assertEquals(0, manager.getAllDirectModulesStartingWith("ab").size());
     }
 
+    public void testBuildKeysForRegularEntries() {
+        ProjectModulesManager modulesManager = (ProjectModulesManager) nature2.getAstManager().getModulesManager();
+        String pythonpath = TestDependent.TEST_PYSRC_TESTING_LOC + "\\testWoInit\\";
```

Comment:
I don't see `testWoInit` in the commit. Also, folders should be lowercase (i.e.: test_wo_init). In this case, I think a better name would be `namespace_pkg` (as it's what we're checking).

Also, please use `/` instead of `\\` so that it works in Linux and Windows.